src/pages/registration/components/RegistrationNotFound.tsx

```diff
@@ -0,0 +1,39 @@
+import { Container, Heading, Flex, Text, Divider } from '@chakra-ui/layout';
+import { Button, Box } from '@chakra-ui/react';
+import { HiExternalLink } from 'react-icons/hi';
+import { useParams } from 'react-router';
+
+import { useDarkMode } from 'lib/hooks/useDarkMode';
+import { getReadableTerm } from 'lib/utils/terms';
+
+export function RegistrationNotFound() {
+  const { term } = useParams();
+  const mode = useDarkMode();
+  return (
+    <Container maxW="container.xl" centerContent data-testid="registration-component-1">
+      <Divider my="4" />
+      <Box padding="10">
+        <Flex direction={{ md: 'row', base: 'column' }} justifyContent="space-between">
+          <Heading size="md" color={mode('gray', 'dark.header')}>
+            Unable to find saved courses from{' '}
+            <Text as="span" color={mode('black', 'white')}>
+              {getReadableTerm(term)}&nbsp;
+            </Text>
+            timetable
+            <Box padding="4">
+              <Button
+                colorScheme="blue"
+                width="auto"
+                leftIcon={<HiExternalLink />}
+                as="a"
+                href={`https://courseup.vikelabs.dev/scheduler/${term}`}
```

Comment:
```suggestion
                as={Link}
                to={`/scheduler/${term}`}
```
And you will need to import `Link` from react router (docs on this https://v5.reactrouter.com/web/api/Link)
Alos just an FYI, `courseup.vikelabs.dev` is our staging environment so we wouldn't want to link our users there anyways